File: distros/L/LLEval/example/fizzbuzz.js
#!lleval
for (var i = 0; i <= 30; i++){
    print (
        i % 15
            ? i % 5
                ? i % 3
                    ? i : 'Fizz'
                : 'Buzz'
            : 'FizzBuzz'
    );
}

File: distros/L/LLEval/example/fizzbuzz.hs
#!lleval
main = mapM_ (putStrLn . fizzbuzz) [1..30]

fizzbuzz n
   = if or $ map (\s->length s /= 0) [fizz, buzz] 
        then fizz ++ buzz 
        else show n
     where fizz = if mod n 3 == 0 then "Fizz" else ""
           buzz = if mod n 5 == 0 then "Buzz" else ""

